dog.py

```diff
@@ -49,16 +51,88 @@
 SUDO_OUTSIDE_DOCKER = 'sudo-outside-docker'
 TERMINAL = 'terminal'
 UID = 'uid'
+USB_DEVICES = 'usb-devices'
 USER = 'user'
 USER_ENV_VARS = 'user-env-vars'
 USER_ENV_VARS_IF_SET = 'user-env-vars-if-set'
+USE_PODMAN = 'use-podman'
 VERBOSE = 'verbose'
+VERSION = 'version'
 VOLUMES = 'volumes'
 WIN32_CWD = 'win32-cwd'
 
 DogConfig = Dict[str, Union[str, int, bool, Path, List[str], Dict[str, str]]]
 
 
+DEFAULT_CONFIG = {
+    ARGS: ['id'],
+    AS_ROOT: False,
+    AUTO_MOUNT: True,
+    CWD: '/home/nobody',
+    EXPOSED_DOG_VARIABLES: [UID, GID, USER, GROUP, HOME, AS_ROOT, VERSION],
+    GID: 1000,
+    GROUP: 'nogroup',
+    HOME: '/home/nobody',
+    HOSTNAME: 'dog_docker',
+    INTERACTIVE: True,
+    PORTS: {},
+    PULL: False,
+    SANITY_CHECK_ALWAYS: False,
+    SUDO_OUTSIDE_DOCKER: False,
+    TERMINAL: False,
+    UID: 1000,
+    USB_DEVICES: {},
+    USER: 'nobody',
+    USER_ENV_VARS: {},
+    USER_ENV_VARS_IF_SET: {},
+    USE_PODMAN: False,
+    VERBOSE: False,
+    VERSION: DOG_VERSION,
+    VOLUMES: {},
+}
+
+
+class UsbDevices:
```

Comment:
A shame that we have to find the USB full path - but i guess that there is no other way around.

Reply:
No, I agree. This is the only fast way of translating a vendor:product to a "device" path on Linux, I could find. And that is needed as it depends on which port the device is plugged into.
An alternative would be an external script but having this as part of dog.config is really nice.